packages/kbn-i18n/README.md

```diff
@@ -252,32 +252,64 @@ and added as a comment next to translation message at `defaultMessages.json`
 
 
 #### Attributes translation in React
-React wrapper provides an API to inject the imperative formatting API into a React
-component by using render callback pattern. This should be used when your React
-component needs to format data to a string value where a React element is not
-suitable; e.g., a `title` or `aria` attribute. In order to use it, you should
-wrap your components into `I18nContext` component. The child of this component
-should be a function that takes `intl` object into parameters:
+
+React wrapper provides an ability to inject the imperative formatting API into a React component via its props using `injectI18n` HOC. This should be used when your React component needs to format data to a string value where a React element is not suitable; e.g., a `title` or `aria` attribute. In order to use it you should wrap your component into `injectI18n` function. The formatting API will be provided to the wrapped component via `props.intl`.
```

Comment:
__nit:__ `HOC` --> `higher-order component (HOC) factory`
__nit:__ ``should wrap your component into `injectI18n` function`` --> ``should wrap your component with `injectI18n` factory``